src/Renderer.hpp

```diff
@@ -51,11 +54,6 @@ class Renderer {
   VkCommandPool graphics_command_pool_;
```

Comment:
Shall we move `swapchain_image_views_` and `swapchain_images_` inside Swapchain class just as project 5 code does?

Reply:
For some reason the image views still reside in the rendered class in project 5

Reply:
For some reason you have `SwapChain::vkSwapChainImages` but you neither created it nor used it, and we're still using `Renderer::swapchain_images_` to create frame resources. Could you fix this?